mithril-aggregator/src/commands/genesis_command.rs

```diff
@@ -73,7 +73,10 @@ impl ExportGenesisSubCommand {
             self.target_path.display()
         );
         let mut dependencies_builder = DependenciesBuilder::new(config.clone());
-        let dependencies = dependencies_builder.create_genesis_container().await?;
+        let dependencies = dependencies_builder
+            .create_genesis_container()
+            .await
+            .with_context(|| "Dependencies Builder can not create genesis container")?;
```

Comment:
`genesis container` is vague when out of context. `cannot create genesis command dependencies container` would imho suit better.